certificate-policy.md

```diff
@@ -1264,15 +1276,28 @@ Subscriber Certificates issued for delegated OCSP responders SHALL have a Validi
 
 ### 6.5.1 Specific computer security technical requirements
 
-The CA SHALL enforce multi-factor authentication for all accounts capable of directly causing certificate issuance.
-
-For all CAs operating under this policy, the computer security functions listed below are required. These functions may be provided by the operating system, or through a combination of operating system, software, and physical safeguards. The CA and its ancillary parts shall include the following functionality:  
+Administrator privileges to all Certificate System components SHALL only be granted to the Administrator trusted role. The CA SHALL enforce multi-factor authentication using a unique credential created by or assigned to the trusted role for all accounts capable of directly causing certificate issuance or authenticating to Certificate Systems. Multifactor authentication SHALL be implemented on all Certificate System components. If multifactor authentication is not supported or is not technically feasible to implement, the following username and password controls, where technically feasible, SHALL be implemented:
+
+- For accounts that are not publicly accessible (accessible only within a Secure or High Security Zone), require passwords have at leaast twelve (12) characters.
```

Comment:
Remove the password allowance. 

Even though the network security requirements "allow" it, USG does not and should not.  These would be privileged accounts and USG requires 2FA for privileged accounts everywhere.    :+1: